src/prevector.h

```diff
@@ -213,11 +226,8 @@ class prevector {
         if (capacity() < n) {
             change_capacity(n);
         }
-        while (first != last) {
-            _size++;
-            new(static_cast<void*>(item_ptr(size() - 1))) T(*first);
-            ++first;
-        }
+        _size += n;
```

Comment:
Seems `n` here should be `difference_type` as well.

Reply:
@Empact Thank you for your review.
However, I think I will close this PR  after a while because of having cherry-pickked by #12549 .